### ISSUE TITLE
Question with training stage 2 

### ISSUE THREAD

**Issue by jinny960812** [Doubiiu/CodeTalker#9]
Hi, I tried to train stage 2 on my dataset, but the loss only oscillates and doesn't go down. 
Should I reduce the learning rate (it is currently 1e-4) ? 
Or should I reset the weight between loss_motion and loss_reg? 
How much should the loss go down? 
Do you have any tips for stage 2 training? 
Thank you!
